x-pack/plugins/ml/public/application/routing/breadcrumbs.ts

```diff
@@ -57,11 +57,25 @@ export const DATA_VISUALIZER_BREADCRUMB: ChromeBreadcrumb = Object.freeze({
 
 export const AIOPS_BREADCRUMB: ChromeBreadcrumb = Object.freeze({
   text: i18n.translate('xpack.ml.aiopsBreadcrumbLabel', {
-    defaultMessage: 'AIOps',
+    defaultMessage: 'AIOps Labs',
   }),
   href: '/aiops/explain_log_rate_spikes_index_select',
 });
 
+export const EXPLAIN_LOG_RATE_SPIKES: ChromeBreadcrumb = Object.freeze({
+  text: i18n.translate('xpack.ml.aiops.explainLogRateSpikesBreadcrumbLabel', {
+    defaultMessage: 'Explain Log Rate Spikes',
+  }),
+  href: '/aiops/explain_log_rate_spikes_index_select',
+});
+
+export const LOG_PATTERN_ANALYSIS: ChromeBreadcrumb = Object.freeze({
```

Comment:
Having run one analysis, if I then click on the 'AIOps Labs' breadcrumb

<img width="432" alt="image" src="https://user-images.githubusercontent.com/7405507/190421985-dfb05db7-35af-4403-a114-d2b682eebeed.png">


it switches to the select data view route for the Explain Log Rate Spikes page rather than remaining in the Log Pattern Analysis page:


<img width="608" alt="image" src="https://user-images.githubusercontent.com/7405507/190421829-22a51090-d8cd-4683-96dd-0a2345e31269.png">